client/src/components/RatingsAndReviews/AddReviewModal.jsx

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
+import ReviewThumbnailContainer from './ReviewThumbnailContainer.jsx';
+import WordCount from './WordCount.jsx';
 
 class AddReviewModal extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      nickname: 'Example: jackson11!',
+      email: '',
+      selectedImageFiles: [],
+      filesLoaded: 0,
+      reviewSummary: 'Example: Best purchase ever!',
+      reviewBody: 'Why did you like the product or not?',
+      characteristics: [],
+      recommendation: 'yes',
+      overalRating: 0
```

Comment:
nit but is it overallRating vs overalRating, and is it used?

Reply:
Good catch. It mirrors the review/ratings info that will be passed in to the component, but I suppose it could just be called 'rating' since it will be populated by this one review.